vulnfetcher/ossvulnfetcher/osindexfetcher.go

```diff
@@ -49,53 +54,121 @@ func (n *OSSIndexFetcher) Fetch() error {
 	return nil
 }
 
-// Test tests for a package
-func (n *OSSIndexFetcher) Test(name string, version string) ([]vulnfetcher.Vulnerability, error) {
-	request := &OSSPackageRequest{Pm: "npm", Name: name}
-	data, err := json.Marshal([]*OSSPackageRequest{request})
+// BuildCoordinate builds the "coordinates" of an npm package according to its name and version
+func BuildCoordinate(name string, version string) string {
+	var namespace = strings.Replace(name, "@", "", -1)
+	var data = "npm:" + namespace + "@" + version
+	return data
+}
+
+// ParseCVEFromTitle parses CVE identifier from title field ( it used to be a dedicated field in API v2, in API v3 it must be parsed...)
+func ParseCVEFromTitle(title string) string {
+	re := regexp.MustCompile("^\\s*\\[(CVE.*?)\\]")
+	res := re.FindStringSubmatch(title)
+
+	if len(res) != 2 {
+		return ""
```

Comment:
I guess erroring here is not right as it would stop the show right?

Reply:
exactly :white_check_mark: not all titles containe a CVE:
- some contain a CWE instead (which is parsed in #15)
- some don't contain any kind of id of the vulnerability/weakness described